src/arrays/programs_involving_multidimensional_data/medium/q5/MainTest.java

```diff
@@ -0,0 +1,30 @@
+package arrays.programs_involving_multidimensional_data.medium.q5;
+
+import global.BaseRandomTest;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+
+public class MainTest extends BaseRandomTest {
+    // Parsons with Distractors
+
+    public static final int n = 2;
+    public static final int m = 13;
+    public static final char start = 97;
+
+    public Clause[] testSentence() {
+        Clause[] c = new Clause[n * m + n];
```

Comment:
You could probably add an incorrect structure error message for this question because it is just one test.